global.json

```diff
@@ -1,3 +1,3 @@
 {
-  "projects": [ "src" ]
+  "projects": [ "src", "tests" ]
```

Comment:
I think we can undo this as well, it was only for the BenchmarkDotNet stuff, right?